cmd/lncli/commands.go

```diff
@@ -719,6 +719,11 @@ var closeChannelCommand = cli.Command{
 			Usage: "the output index for the funding output of the funding " +
 				"transaction",
 		},
+		cli.StringFlag{
+			Name: "chan_point",
+			Usage: "(optional) the channel point. If set, funding_txid " +
+				"and output_index flags will be ignored",
```

Comment:
Should also mention that any positional arguments will also be ignored.

Reply:
corrected